packages/ui/src/app/pages/Election/BlacklistedAccounts/BlacklistedAccounts.tsx

```diff
@@ -0,0 +1,120 @@
+import BN from 'bn.js'
+import React, { useEffect, useMemo, useState } from 'react'
+import styled from 'styled-components'
+
+import { useBalances } from '@/accounts/hooks/useBalance'
+import { useVotingOptOutAccounts } from '@/accounts/hooks/useVotingOptOutAccounts'
+import { Balances } from '@/accounts/types'
+import { PageHeaderRow, PageHeaderWrapper, PageLayout } from '@/app/components/PageLayout'
+import { InfoSymbol } from '@/common/components/icons/symbols'
+import { MainPanel, RowGapBlock } from '@/common/components/page/PageContent'
+import { PageTitle } from '@/common/components/page/PageTitle'
+import { Pagination } from '@/common/components/Pagination'
+import { TokenValueStat } from '@/common/components/statistics'
+import { TextBig, TextInlineMedium } from '@/common/components/typography'
+import { Warning } from '@/common/components/Warning'
+
+import { ElectionTabs } from '../components/ElectionTabs'
+
+import { BlacklistedAccount } from './BlacklistedAccount'
+
+interface TotalBalanceProp {
+  addresses: string[]
+}
+
+export const BlacklistedAccounts = () => {
+  const ACCOUNTS_PER_PAGE = 18
+  const [page, setPage] = useState(1)
+  const votingOptOutAccounts = useVotingOptOutAccounts()
+  const [paginatedAccounts, setPaginatedAccounts] = useState<string[] | undefined>()
```

Comment:
As a result `useBalances` should be called on this component instead. Also I think the sum can be done here (no need for `TotalBalance`), and the paginated accounts should be `{ address: string, balance: BN | undefined }`:
```suggestion
  const balances = useBalances(addresses)
  const totalBalance = useMemo(
    () => balances.values().reduce((prev, value) => value?.total.add(prev) ?? prev, BN_ZERO),
    [balances]
  )
  const paginatedAccounts = useMemo(
    () =>
      votingOptOutAccounts
        .slice((page - 1) * ACCOUNTS_PER_PAGE, page * ACCOUNTS_PER_PAGE)
        .map((address) => ({ address, balance: balance.get(address)?.total }))
    [votingOptOutAccounts, page, balances])
  )
```